tba-unit-tests/View Elements/Match/MatchViewModelTests.swift

```diff
@@ -0,0 +1,138 @@
+import XCTest
+@testable import The_Blue_Alliance
+
+class MatchViewModelTestCase: CoreDataTestCase {
+    func test_no_breakdown() {
```

Comment:
space above please

Reply:
Added in 9f02a264bb7da94be1164f8501639943760201ef